tests/docker/setup.yml

```diff
@@ -0,0 +1,28 @@
+---
+# set ft=ansible
+#
+
+- name: Check for docker-latest package if running against docker-latest
```

Comment:
These two tasks can probably be shortened to a single task when `rpm -q docker-latest` is run and it fails if the RC is 1